cobigen/cobigen-xmlplugin/src/main/java/com/devonfw/cobigen/xmlplugin/inputreader/XmlInputReader.java

```diff
@@ -250,4 +251,11 @@ public Object read(Path path, Charset inputCharset, Object... additionalArgument
         }
         throw new IllegalArgumentException("Currently folders are not supported as Input by XmlInputReader#read");
     }
+
+    @Override
+    public boolean isMostLikelyReadable(Path path) {
+        String validExtension = "xml";
```

Comment:
Can you also add `xmi`? It is an standard for UML diagrams. CobiGen is able to read a class diagram and generate code from it.

Reply:
Done